### ISSUE TITLE
The costs of storage decrease when the losses increase

### ISSUE THREAD

**Issue by MartLubben** [quintel/etmodel#3246]
When the heat losses of storage are increased less heat is sold. The costs of the heat storage is based on euro/MWh of sold heat. As a result the costs decrease when the loss increases. When making a scenario the cost input should take into account the losses, but for a user this is not so obvious. 

![Webp net-gifmaker (3)](https://user-images.githubusercontent.com/33415156/75561538-79270680-5a47-11ea-9ba3-cf5a0badd769.gif)

**Comment by michieldenhaan** [comment]
Fair point, I'll add a note to the slider description to remind the user to change storage costs as well when changing losses.

Ideally, this should happen automatically in the future. E.g. we could specify investment + O&M costs of storage rather than a fixed cost per MWh. However, it is currently not supported to calculate a number of units for the storage node, so we cannot specify investment costs per unit.

**Comment by ChaelKruip** [comment]
I think this is a valuable point still.
